designs/webviews/webviews-general.md

```diff
@@ -0,0 +1,95 @@
+# Webviews for Rich UIs in VS Code
+
+## Objective
+
+Justify expanding the criteria for using webviews within the AWS Toolkit for VS Code in order to create richer UIs than VS Code natively allows.
```

Comment:
"Justify" doesn't seem appropriate for a design doc. Suggest:

"Guidance for when to use webviews to enhance UX beyond the UI constructs provided by VS Code."

Reply:
Link to https://code.visualstudio.com/api/extension-guides/webview the first mention of "webview" in the doc.